source/includes/api/_before_you_get_started.md

```diff
@@ -1,27 +1,60 @@
 # Before you get started
 
-This section covers a list of topics you should be familiar with before using the API. These concepts span the whole API.
+This section covers a list of topics you should be familiar with before using the API. The concepts described in this section span across multiple API endpoints and are fundamental for a better understanding of how to interact with the RW API.
 
 ## Applications
 
-TODO
+As you might have come across while reading these docs, different applications and websites rely on the RW API as the principal source for their data. While navigating through the catalog of available datasets, you will find some datasets used by the [Resource Watch website](https://resourcewatch.org/), others used by [Global Forest Watch](https://www.globalforestwatch.org/). In many cases, applications even share the same datasets!
 
+To ensure the correct separation of content across the different applications that use the RW API, you will come across a field named `application` in many of the API's resources (such as datasets, layers, widgets, and others). Using this field, the RW API allows users to namespace every single resource, so that it's associated only with the applications that use it.
 
-## Authentication
+### Officially supported applications
 
-TODO
+Currently, the following applications are using the API as the principal source for their data:
 
+* the [Resource Watch website](https://resourcewatch.org/), where the `application` field takes the value `rw`;
+* the [Global Forest Watch website](https://www.globalforestwatch.org/), where the `application` field takes the value `gfw`;
+* the [Partnership for Resilience and Preparedness website](https://prepdata.org/), where the `application` field takes the value `prep`;
+* the [Forest Atlases websites](https://www.wri.org/our-work/project/forest-atlases) for different countries of the world also rely on the RW API - in the case of these websites, the `application` field takes the value `forest-atlas`;
 
-## Roles
+However, please keep in mind that, in most API resources, the values provided for the `application` field are not subject to validation. Thus, feel free to identify your applications in whatever way you prefer when creating content for your applications. --> TODO: check if we actually want to broadcast this...?
```

Comment:
I would not document this because, as far as I am concerned, the only reason this is so is because I haven't had time to change it :P IMO, ideally, we should have a central list of applications, and only privileged (ADMIN? ) users could add new entries to it.